packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -0,0 +1,434 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { assert } from "@fluidframework/common-utils";
+import { Value, Anchor } from "../../tree";
+import {
+	IEditableForest, TreeNavigationResult, mapCursorField, ITreeSubscriptionCursor, ITreeSubscriptionCursorState,
+} from "../../forest";
+import { brand } from "../../util";
+import { TreeSchema, FieldSchema, rootFieldKey, LocalFieldKey, TreeSchemaIdentifier } from "../../schema-stored";
+import { FieldKind, Multiplicity } from "../modular-schema";
+import {
+    AdaptingProxyHandler,
+    adaptWithProxy,
+    getFieldKind, getFieldSchema, getPrimaryField, isPrimitive, isPrimitiveValue, PrimitiveValue,
+} from "./utilities";
+
+// Symbol for extracting target from editable-tree proxies.
+// Useful for debugging and testing, but not part of the public API.
```

Comment:
```suggestion
/**
 * Symbol for extracting target from editable-tree proxies.
 * Useful for debugging and testing, but not part of the public API.
 */
```

Reply:
I'm a little confused by the name: if this is used to access the target of the proxy then shouldn't it named `target` or `proxyTarget`? This is not used to access the proxy, right?